docs/sql-data-sources-avro.md

```diff
@@ -240,6 +240,14 @@ Data source options of Avro can be set via:
     </td>
     <td>function <code>from_avro</code></td>
   </tr>
+  <tr>
+    <td><code>writerSchema</code></td>
+    <td>None</td>
+    <td>Optional Avro schema (in JSON format) that was used to serialize the data. This should be set if the schema provided
+      for deserialization is compatible with - but not the same as - the one used to originally convert the data to Avro.
+    </td>
```

Comment:
Would it be possible to link to the Confluent documentation? They have an excellent document on schema compatibility and evolution: https://docs.confluent.io/current/schema-registry/avro.html